### DISCUSSION TITLE
Applying Settings from Czkawka Gui?

### DISCUSSION THREAD

**Discussion by terminalwhoami** [jlesage/docker-czkawka#27] [Q&A] [answered]
I changed some settings in Czkawka's GUI and the GUI now says I need to restart the app to apply settings.  How do I do that?

Here's what I've tried:
stopped and restarted the app from TrueNas GUI
changed some settings in the the TrueNAS GUI so it would redeploy
Used the shell to kill the process within the container

**Comment by jlesage** [comment, marked as answer]
Restarting the container should do the job.

**Comment by terminalwhoami** [comment]
Does the app gui always have that warning? Maybe it is applying and I’m just assuming it’s not because the warning is there. On Jan 5, 2024, at 5:25 PM, Jocelyn Le Sage ***@***.***> wrote:﻿
Restarting the container should do the job.

**Comment by jlesage** [comment]
What warning are you talking about ?  If you mean the message in the settings window, then yes it is always displayed.

![image](https://github.com/jlesage/docker-czkawka/assets/1791123/b1421f1b-8711-4bf2-8231-87ec49a04834)

**Comment by terminalwhoami** [comment]
Yes, that one. I guess I’m used to seeing a message like that pop up when the app needs to be restarted. So I guess I’m good to go!
